projboard/tests/test_article.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+from projboard.models.article import Article, Like, View, Subject, User
+
+
+TITLE = "Article Test"
+CONTENT = "I'm not gonna write all that!"
+
+
+class TestArticle:
+
+    @pytest.fixture
+    def sup(self):
+        return Subject(name='N')
+
+    @pytest.fixture
+    def sup1(self):
+        return Subject(name='R')
```

Comment:
should move to one fixture
Consider returning [subject1,Subject2]
Please save it here.

In the test itself you should find a way to retrieve the data fast,
Maybe there is a need for helper method in the model for that

Reply:
done